packages/react/src/components/DataGrid/index.tsx

```diff
@@ -143,8 +149,12 @@ export const DataGrid = <T extends Row>({
     if (showEmptyPlaceholder) {
       return (
         <div className="c__datagrid__empty-placeholder fs-h3 clr-greyscale-900 fw-bold">
-          <img src={emptyImageUrl} alt={t("components.datagrid.empty_alt")} />
-          {t("components.datagrid.empty")}
+          {!hideEmptyPlaceholderImage && (
+            <img src={emptyImageUrl} alt={t("components.datagrid.empty_alt")} />
+          )}
+
+          {emptyPlaceholderLabel ?? t("components.datagrid.empty")}
+          {emptyCta}
```

Comment:
Is it not enough to just have 1 prop to keep it easy?
```suggestion
          {emptyNode ?? t("components.datagrid.empty")}
```

Reply:
What I like by having two props is that we can provide only `emptyCta` if we want to just add a button without modifying the default placeholder.

With one only prop we would need to provide the default label each time. 🤔